examples/distributed/pyg/partition_graph.py

```diff
@@ -106,17 +129,32 @@ def get_idx_split(dataset, dataset_name, split_data):
         test_idx = split_idx['test']
         val_idx = split_idx['valid']
 
+    elif dataset_name == 'MovieLens':
+        edge_type = ('user', 'rates', 'movie')
+        train_data, val_data, test_data = T.RandomLinkSplit(
+            num_val=0.1,
+            num_test=0.1,
+            neg_sampling_ratio=0.0,
+            edge_types=[edge_type],
+            rev_edge_types=[('movie', 'rev_rates', 'user')],
+        )(dataset[0])
```

Comment:
Looks like we drop the adjustment of message passing here computed in `train_data`, `val_data` and `test_data`. This is not necessarily a blocker (since I don't necessarily see a good way to fix this for now), but something you should be aware of. Currently, you would like information.

Reply:
Sorry, I didn't get that. What do you mean by "we drop the adjustment of message passing here computed in train_data, val_data and test_data"?

Reply:
The `edge_index` is different across different splits for link prediction tasks (in order to not leak information).

Reply:
I see. What about the solution used in the temporal_link_pred example?: [temporal_link_pred.py #L27](https://github.com/pyg-team/pytorch_geometric/blob/787598715c4f0f89d82ab310372c58ed0cc61308/examples/hetero/temporal_link_pred.py#L27) Maybe we can use that one instead?

Reply:
Yeah, using temporal sampling is usually a good option to resolve this.